game_eggs/steamcmd_servers/sunkenland/egg-sunkenland.json

```diff
@@ -0,0 +1,172 @@
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|numeric",
+            "field_type": "text"
+        },
+        {
+            "name": "Auto Update",
+            "description": "Enable automatic updates on boot",
+            "env_variable": "AUTO_UPDATE",
+            "default_value": "1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|boolean",
+            "field_type": "text"
+        },
+        {
+            "name": "[REQUIRED] World GUID",
+            "description": "The world GUID is the later part of the world save file, for example, in this save file 'YourWorldName~11223344-5566-7788-99AA-BBCCDDEEF F00', the part after '~' symbol is the guid, i.e. '11223344-5566-7788-99AA-BBCCDDEEFF00'. Specify which World to start the game, if the world save file with entered GUID cannot be found, the server will fail to start and shut down itself.",
+            "env_variable": "WORLD_GUID",
+            "default_value": "76c82b71-6e5b-43e1-96af-6d6024a0b756",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Name",
+            "description": "Name of the server",
+            "env_variable": "SERVER_NAME",
+            "default_value": "Pterodactyl World",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Password",
+            "description": "Server password (if any)",
+            "env_variable": "SERVER_PASSWORD",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Region",
+            "description": "Set the custom Server Region. Default is ASIA. Possible values: asia, cn, jp, eu, sa, kr, us, usw",
+            "env_variable": "SERVER_REGION",
+            "default_value": "asia",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Max Players",
+            "description": "Set the Max Player Count. The minimum value is 3 and the maximum value is 15. Default is 3.",
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "3",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|integer|min:3",
+            "field_type": "text"
+        },
+        {
+            "name": "[SYSTEM] WINETRICKS_RUN",
+            "description": "",
+            "env_variable": "WINETRICKS_RUN",
+            "default_value": "mono vcrun2019",
```

Comment:
use vcrun2022. This contains 2015 till 2022

Is Mono really needed?

Reply:
I'll try without and see what happens. Sometimes it's needed for Unity games

Reply:
> I'll try without and see what happens. Sometimes it's needed for Unity games

its only needed, when dotnet is needed. and then use dotnet7 or dotnet8

Reply:
Works fine without `mono` 👍